interface/modules/custom_modules/oe-module-weno/templates/weno_fragment.php

```diff
@@ -162,17 +215,32 @@ function getProviderByWenoId($external_id, $provider_id = ''): string
                 <?php } ?>
             </select>
         </div>
+        <div class="form-group">
+            <label role="button" id="label-location" class="text-primary mb-1 mr-1" for="facilitySelect" title="<?php echo $titleLocation ?>" data-toggle="popover" data-content="<?php echo $popoverLocation ?>">
```

Comment:
I couldn't figure out where $titleLocation is. Just need to ensure it is escaped via attr()